tests/isotonic_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Isotonic Regression with PAV tests."""
+
+from absl.testing import absltest
+
+import jax
+import jax.numpy as jnp
+import numpy as onp
+
+from jax.test_util import check_grads
+from jaxopt._src.isotonic import isotonic_l2_pav
+from jaxopt._src import test_util
+
+
+class IsotonicPavTest(test_util.JaxoptTestCase):
+
+    def test_output_shape_and_dtype(self):
+        n = 10
+        y = jax.random.normal(jax.random.PRNGKey(0), (n,))
+        output =  isotonic_l2_pav(y)
+        assert output.shape == y.shape
+        assert output.dtype == y.dtype
+
+    def test_gradient(self):
```

Comment:
```suggestion
    def test_gradient(self, n = 10):
```
and remote the line below. This way its clear that the test could be run with other values of n

Reply:
same for the test below